src/main.cpp

```diff
@@ -64,11 +64,11 @@ int main(int argc, char *argv[])
         std::cout << "Maximum number of molluscs: " << maxNumOfMolluscs << std::endl;
     }
 
-    auto useCam = true;
+    bool useCam;
```

Comment:
This is both ugly and wrong.  
First, we try to use right-hand coding style combined with auto (yes, I know, we don't do this everywhere).  
Second, you're invoking undefined behavior here - there is [no default value for non-class vars](https://en.cppreference.com/w/cpp/language/default_initialization), and even if there were, it would probably be false.  
TL;DR: Please don't do this.